src/3rdparty/walkontable/src/overlay/top.js

```diff
@@ -320,6 +320,7 @@ export class TopOverlay extends Overlay {
       return this.wot.wtTable.holderOffset.top;
 
     }
+
     return 0;
 
```

Comment:
Awesome!

We may also use negative rule `never` to fix new lines after `return`:  `{ blankLine: "never", prev: "return", next: "*" }`

Reply:
This one actually won't work as the ESLint didn't see any statement or so after the `return` keyword. So it can't remove the new line :(

Reply:
Ok. Maybe they have some other rules for this. But lets keep it simple, one at a time